bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/ip.xml

```diff
@@ -64,6 +72,32 @@
 				<description>Seconds between connection retries when KNX link has been lost, 0 means never retry, minimum 30s</description>
 				<default>60</default>
 			</parameter>
+			<parameter name="routerBackboneKey" type="text" groupName="knxsecure">
```

Comment:
Are those parameters hidden when the group is hidden? Or is it necessary to also mark them as `advanced`? It would imho be cleaner.

Reply:
Good point. Actually, it did not properly work to set "advanced" on group level.
I moved "advanced" down to the elements and added a note to the SECURE settings `<option value="SECURETUNNEL">Secure tunnel (experimental, use advanced options to configure)</option>`
Cleaner.